quixstreams/app.py

```diff
@@ -657,7 +686,9 @@ def get_producer(self) -> Producer:
             extra_config=self._producer_extra_config,
         )
 
-    def get_consumer(self, auto_commit_enable: bool = True) -> Consumer:
+    def get_consumer(
```

Comment:
Just curious as to why this was changed?

Reply:
also extra "m" in  `consumer_group` =)

Reply:
This can be reverted. Some left over of a test I forgot to remove :facepalm: